src/integrations/buttons.js

```diff
@@ -141,6 +141,17 @@ function addHostCopyNetworksButton () {
   })
 }
 
+function addHostCpuPinningButton () {
+  getPluginApi().addDetailPlaceActionButton(entityTypes.host, entityTypes.hostDevices, msg.cpuPinningModalButton(), {
```

Comment:
don't we need to add a panel there (Host/HostDevices) for this?

Reply:
we already did: https://github.com/oVirt/ovirt-engine/pull/87